services/secret-service/src/middleware/auth/index.js

```diff
@@ -15,11 +16,9 @@ async function userIsOwnerOf(dao, req, res, next) {
         if (!doc) {
             return next({ status: 404 });
         }
-        const userIsOwner = doc.owners.find(
-            elem => elem.id === req.user.sub,
-        );
+        const userHasAccess = (validation && validation(doc)) || doc.owners.find((elem) => elem.id === req.user.sub);
```

Comment:
Is it redundant to have the `doc.owners.find` section of the statement, because this should already be covered by a similar statement inside `isOwnerOf()` in _iam-utils_?
https://github.com/openintegrationhub/openintegrationhub/blob/e8b1354c0819debaa7b6516fc82f2f38a05673cc/lib/iam-utils/src/index.js#L273-L274
Or did you have a case where this was needed?

Reply:
No, you were right. It was a redundant statement. We can use there directly the middleware from iam-utils just with the variable of the allTenantUsers.